aptiche/lib/widgets/HomeScreen/drawer_list_tile.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+class DrawerListTile extends StatelessWidget {
+  const DrawerListTile({
+    Key? key,
+    required this.text,
+  }) : super(key: key);
+  final String text;
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 10.0, bottom: 15.0),
+      child: ListTile(
+        title: Text(
+          text,
+          style: Theme.of(context).primaryTextTheme.bodyText2!.copyWith(
+                fontWeight: FontWeight.bold,
+                fontSize: 20,
```

Comment:
dynamic sizes here as well

Reply:
Font sizes would be in this way only right. Size config here also? We didn't use them in theme defining na.